messages.xlf

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I think this file was added by error, translations are in src/locale/

Reply:
Es bueno tenerlo, porque es el que se genera automáticamente y sirve de inventario para agregar las nuevas traducciones a los archivos para Argentina y Brasil